Parse/src/main/java/com/parse/ManifestInfo.java

```diff
@@ -462,10 +461,6 @@ private static boolean hasAnyGcmSpecificDeclaration() {
     return false;
   }
 
-  private static boolean isGooglePlayServicesAvailable() {
-    return Build.VERSION.SDK_INT >= 8 && getPackageInfo("com.google.android.gsf") != null;
```

Comment:
Can we simplify this PR by deleting these two lines here and replacing it with the code you wrote for `isGcmRegisterServiceAvailable`?